infra/ci/build.py

```diff
@@ -87,8 +87,34 @@ def check_build(project, engine, sanitizer, architecture):
   ])
 
 
+def should_build_coverage(project_yaml):
+  """Returns True if a coverage build should be done based on project.yaml
+  contents."""
+  # Enable coverage builds on projects that use engines. Those that don't use
+  # engines shouldn't get coverage builds.
+  engines = project_yaml.get('fuzzing_engines', DEFAULT_ENGINES)
+  engineless = 'none' in engines
+  if engineless:
+    assert_message = 'Project supports engines and none: {engines}.'.format(
```

Comment:
This message seems a bit confusing.

Consider rewording along the lines of `'"none" engine must be the only one specified.'`

Reply:
Done.